src/components/Modal/Modal.stories.jsx

```diff
@@ -16,9 +16,4 @@ storiesOf('Modal', module)
       <Modal show formType={false} />
     </Provider>
```

Comment:
Remove the provider and write the story for the main component, not the connected one

Reply:
It throws errors without the provider. I will attempt to work around it but then, it was because of the errors I discovered the use of the store in storybook.

Reply:
Export the main component as a named export and pass in whatever the component needs as a prop of action. Export the connected component as a default export for the app's usage. You can ask any of your peers if you hit any snag during this implementation.

Reply:
I know about that and it has its limit